includes/docker/docker-compose.yml

```diff
@@ -1,38 +1,48 @@
-version: '2.1'
+# This docker-compose.yml file was provided by Toolkit. When using the docker
+# environment you will have to update your build.develop.props file to change
+# the connection settings accordingly:
+#
+# # Example of build.develop.props file for usage of this docker-compose.yml.
+# # ============================================================================
+# db.host = mysql
+# solr.host = solr
+# behat.base_url = http://web:8080/build
+# behat.wd_host.url = http://selenium:4444/wd/hub
+# project.url.base = https://<the-url-on-which-you-will-access-your-project>/build
+
+version: '2'
 services:
   web:
-    image: fpfis/php56-dev
-    build: ./docker-php
-    environment:
-      DOCUMENT_ROOT: ${WORKSPACE}/build
-      COMPOSER_CACHE_DIR: /var/cache/composer
-      COMPOSE_PROJECT_NAME: ${COMPOSE_PROJECT_NAME}
+    image: fpfis/httpd-php-dev:5.6
+    working_dir: ${PWD}
     ports:
-      - 8080
+      - 8080:8080
     volumes:
-      - ${WORKSPACE}:${WORKSPACE}
-      - ${WORKSPACE_CACHE:-/data/cache}:/tmp/toolkit
-      - ${WORKSPACE_CACHE:-/data/cache}:/cache
-      - /var/cache/composer:/var/cache/composer
-      - /usr/bin/docker:/usr/bin/docker
-      - /usr/bin/docker-compose:/usr/bin/docker-compose
-      - /var/run/docker.sock:/var/run/docker.sock
-    working_dir: ${WORKSPACE}
-    links:
-      - mysql:mysql
-      - selenium:selenium
-    expose:
-      - '8080'
+      - ${PWD}:${PWD}
+    environment:
+      DOCUMENT_ROOT: ${PWD}
+      GITHUB_API_TOKEN:
+      XDEBUG_CONFIG: remote_connect_back=0 remote_host=172.17.0.1 remote_enable=1
   mysql:
-    image: fpfis/mysql56
-  #redis:
-  #  image: fpfis/redis
-  #varnish:
-  #  image: fpfis/nexteuropa-varnish5
+    image: percona/percona-server:5.6
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+    volumes:
+      - mysql:/var/lib/mysql
+  # The following solr cores are available:
+  # - http://127.0.0.1:8983/solr/d7_search_api
+  # - http://127.0.0.1:8983/solr/d7_apachesolr
   solr:
-    # The following cores are available :
-    # - http://127.0.0.1:8983/solr/d7_search_api
-    # - http://127.0.0.1:8983/solr/d7_apachesolr
     image: fpfis/solr5
+  # For integration with nexteuropa_varnish please read the documentation here:
+  # https://github.com/ec-europa/nexteuropa-vcl#drupal-7-nexteuropa-projects
+  varnish:
+    image: fpfis/varnish-ne:4.1
   selenium:
-    image: selenium/standalone-chrome
+    image: selenium/standalone-chrome-debug
```

Comment:
What do you say about keep this aligned with the one provided in the pipele [here|https://github.com/ec-europa/ne-pipelines/blob/master/drone/test/toolkit.yml#L289] that is currently using `image: selenium/standalone-chrome:3.141.5`

Reply:
Proposing to keep the latest selenium version on this docker-compose.yml file. The versions in Drone pipelines were fixed because of this issue: https://webgate.ec.europa.eu/CITnet/jira/browse/MULTISITE-21493

But we have no real track record of which projects were affected by this. So I think it's best to keep it on latest to find out if any projects are having issues with the latest version.